demos/serverless/deploy.js

```diff
@@ -46,6 +46,18 @@ const supportedControlRegions = [
   'us-gov-west-1',
 ];
 
+let useChimeSDKMediaPipelines = true;
+let chimeSDKMediaPipelinesEndpoint = 'https://media-pipelines-chime.us-east-1.amazonaws.com';
```

Comment:
[Checking] Is this a public facing endpoint we want to keep outside?

We have few endpoints not intended to show in public repo which are stored as GitHub secrets.

Reply:
thanks for checking. That is a public facing endpoint: https://docs.aws.amazon.com/chime-sdk/latest/dg/migrate-pipelines.html